docs/static/breaking-changes-90.asciidoc

```diff
@@ -86,3 +86,11 @@ removed and their replacements.
 |=======================================================================
 
 ====
+
+[discrete]
+[[enterprise_search-deprecated-9.0]]
+===== Deprecation Notice: App Search and Workplace Search Plugins
+
+The Enterprise Search product is deprecated and will not be included in the Elastic Stack 9.x release.
+As a result, the App Search and Workplace Search plugins are also deprecated. These plugins will only receive security updates and critical bug fixes moving forward.
+For more details, visit: https://www.elastic.co/guide/en/enterprise-search/current/app-search-workplace-search.html
```

Comment:
@flexitrev Is https://www.elastic.co/guide/en/enterprise-search/current/app-search-workplace-search.html the only page to point user about the product deprecation and transition? The instruction is a bit vague